### PR TITLE
Change the download button CTA for non-Fx users. (bug 1170736)

### PR DESCRIPTION
**Pull request by FrancescoSTL**
I changed the button which is shown to link users to download Firefox directly rather than the 'learn more' popup CTA. This change is reflected by A/B testing done previously and documented by bug 1170736.

@magopian r?

### CODE REVIEW

File: static/css/impala/buttons.less [hunk 1]
```diff
@@ -304,3 +304,36 @@ button[disabled],
 .button.caution.concealed {
     pointer-events: none;
 }
+.button.CTA {
+    display:inline-block;
+    *display:inline;
+    *zoom:1;
```
Comment (magopian) [inline static/css/impala/buttons.less]: Those two are for supporting old versions of IE, right? If so, could you please add a comment?

File: static/css/impala/buttons.less [hunk 2]
```diff
@@ -304,3 +304,36 @@ button[disabled],
 .button.caution.concealed {
     pointer-events: none;
 }
+.button.CTA {
+    display:inline-block;
```
Comment (magopian) [inline static/css/impala/buttons.less]: We usually add a space after the colon.

File: static/css/impala/buttons.less [hunk 3]
```diff
@@ -304,3 +304,36 @@ button[disabled],
...
+    *display:inline;
+    *zoom:1;
+    border-radius:.25em;
+    box-shadow:0 1px 0 0 rgba(0,0,0,0.2),inset 0 -1px 0 0 rgba(0,0,0,0.3);
```
Comment (magopian) [inline static/css/impala/buttons.less]: We usually add spaces after comas.

**Comment by magopian** [comment]
We usually try to order the css properties, not sure if they are atm, nor in which order. Maybe try to group properties that are alike (i'm thinking about the `ms-filter` ones, the text ones, the colors...).

Also, could you please add some screenshots to this PR taken on the major browsers if possible? This would help get a feel of how the end result looks like.

Thanks!

**Comment by FrancescoSTL** [comment]
Firefox spoofed as IE:
screen shot 2015-06-08 at 10 45 34 am

Safari:
screen shot 2015-06-08 at 10 45 13 am

Chrome:
screen shot 2015-06-08 at 10 44 55 am

**Comment by FrancescoSTL** [comment]
Additionally, the button was designed to match the button used in the A/B test shown here: 
https://bugzilla.mozilla.org/show_bug.cgi?id=1159335
(see Cmore's screenshot)

**Comment by magopian** [comment]
Didn't get your last comment, do you want to keep the current CTA **off**?
If so, should it be displayed in the other cases (on the home page for example)?

If not (if it's not used anymore anywhere), then maybe you should take it out of the templates and get rid of the js/css attached to it?

**Comment by FrancescoSTL** [comment]
Eventually, yes, the current CTA will need to be removed entirely. For now, it is still being used in the template for the homepage. Once I work on implementing the change in Cmore's AMO homepage A/B test, I will remove the current CTA entirely.

**Comment by magopian** [comment]
Understood, thanks!
r+

**Comment by magopian** [comment]
Merged in https://github.com/mozilla/olympia/commit/1db0172bf4439705ce648e3026d29b29905b53de

**Pull request closed without merging**